<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-OKOQ08C" failures="1" tests="7" name="net.focltng.testcases.T007" time="98.637" errors="0" timestamp="2022-02-09T19:18:09 IST" skipped="6">
  <testcase classname="net.focltng.testcases.T007" name="TC_088" time="98.637">
    <failure message="Cruise selection page is not Loaded expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Cruise selection page is not Loaded expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at net.focltng.testcases.T007.TC_088(T007.java:717)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_088 -->
  <system-out/>
  <testcase classname="net.focltng.testcases.T007" name="TC_090">
    <skipped/>
  </testcase> <!-- TC_090 -->
  <system-out/>
  <testcase classname="net.focltng.testcases.T007" name="TC_089">
    <skipped/>
  </testcase> <!-- TC_089 -->
  <system-out/>
  <testcase classname="net.focltng.testcases.T007" name="TC_069">
    <skipped/>
  </testcase> <!-- TC_069 -->
  <system-out/>
  <testcase classname="net.focltng.testcases.T007" name="TC_086">
    <skipped/>
  </testcase> <!-- TC_086 -->
  <system-out/>
  <testcase classname="net.focltng.testcases.T007" name="TC_019">
    <skipped/>
  </testcase> <!-- TC_019 -->
  <system-out/>
  <testcase classname="net.focltng.testcases.T007" name="TC_071">
    <skipped/>
  </testcase> <!-- TC_071 -->
  <system-out/>
</testsuite> <!-- net.focltng.testcases.T007 -->
